py/setup.py

```diff
@@ -0,0 +1,40 @@
+import os
+import setuptools
+import shutil
+
+def get_requirements():
+    with open(u"requirements.txt") as fp:
+        return [x.strip() for x in fp.read().split("\n") if not x.startswith("#")]
+
+# Copies the schema in the module so that setuptools is able to
+# find the file and add it to the package.
+if os.path.isfile("../relay/event.schema.json"):
+    shutil.copyfile(
+        "../relay/event.schema.json",
+        "./sentry_data_schemas/event.schema.json"
+    )
```

Comment:
This feels like bad practice to me as it clobbers the check out and file system. Do you think we can find a better directory layout instead?